libs/sdk-ui-kit/src/Bubble/BubbleTrigger.tsx

```diff
@@ -14,6 +14,7 @@ export interface IBubbleTriggerProps {
     tagName?: React.ElementType;
     onBubbleOpen?: () => void;
     onBubbleClose?: () => void;
+    initOpen?: boolean;
```

Comment:
Can we name it `openOnInit` so its same as in other components that have similar prop? (date filter, dropdown, attribute filter)